devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -408,13 +408,13 @@ def configure() {
 
 	if (device.getDataValue("manufacturer") == "SmartThings") {
 		log.debug "Refreshing Values for manufacturer: SmartThings "
-		/* These values of Motion Threshold Multiplier(0x01) and Motion Threshold (0x0276)
+		/* These values of Motion Threshold Multiplier(0x01) and Motion Threshold (0x7602)
 		 seem to be giving pretty accurate results for the XYZ co-ordinates for this manufacturer.
 		 Separating these out in a separate if-else because I do not want to touch Centralite part
 		 as of now.
 		*/
 		configCmds += zigbee.writeAttribute(0xFC02, 0x0000, 0x20, 0x01, [mfgCode: manufacturerCode])
-		configCmds += zigbee.writeAttribute(0xFC02, 0x0002, 0x21, 0x0276, [mfgCode: manufacturerCode])
+		configCmds += zigbee.writeAttribute(0xFC02, 0x0002, 0x21, 0x7602, [mfgCode: manufacturerCode])
```

Comment:
This should be changed to "7602". The end result is the same as what you did but I think it's more clear personally. It'd be good to add a comment too to say it's passed as little endian to work around a bug